VideoLocker/src/main/java/org/edx/mobile/model/course/BlockModel.java

```diff
@@ -26,17 +26,17 @@
     @SerializedName("graded")
     public boolean graded;
 
-    @SerializedName("multi_device")
+    @SerializedName("student_view_multi_device")
     public boolean multiDevice;
```

Comment:
This variable name should also be changed to reflect the field name too, I guess.